ccd-definition/ComplexTypes/CareSupervision/CancelledHearingBooking.json

```diff
@@ -71,6 +71,36 @@
     "FieldTypeParameter": "HearingPresence",
     "ElementLabel": "In person or remote",
     "SecurityClassification": "Public",
+    "FieldShowCondition": "type=\"DO NOT SHOW\"",
+    "Comment": "Backward compatibility - needed until data migration"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "CancelledHearingBooking",
```

Comment:
Sorry, where is this complex type being used?

Reply:
it is used in fpl-ccd-configuration/ccd-definition/CaseField/CareSupervision/manageHearingsFields.json
It represent hearings that have been adjourned or vacated